spec/acceptance/self_service_spec.rb

```diff
@@ -15,7 +15,7 @@
     # Test Confirms all facts are false which is another indicator the class is performing correctly
     describe 'check no self_service fact is false' do
       it 'if idempotent all facts should be true' do
-        expect(host_inventory['facter']['self_service'].size).to eq(15)
+        expect(host_inventory['facter']['self_service'].size).to eq(16)
```

Comment:
this will need to be 17 now Taika's is merged

Reply:
Looks good!